swagger/instructions.txt

```diff
@@ -0,0 +1,36 @@
+
+Update policies to access SwaggerControler, or add at a path level
+
+    'SwaggerController': {
+		'*': true,
+	},
+    
+Add these routes:
```

Comment:
expecting more clear instructions for adding all the customization.
@gitnanji can we add a section for the how-to authenticate these routes. At least warn about it.